src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1854,28 +1903,53 @@ namespace UnitarySystems {
                         EqSizing.DesHeatingLoad = HeatCapAtPeak;
                     }
                 } else {
-                    HeatCapAtPeak = this->m_DesignHeatingCapacity;
+                    // something seems missing here based on multiple conditional if above
+                    if (this->m_DesignHeatingCapacity != DataSizing::AutoSize) HeatCapAtPeak = this->m_DesignHeatingCapacity;
                 }
             }
             //            if ( ! UnitarySystem( UnitarySysNum ).CoolCoilExists )DXCoolCap = HeatCapAtPeak;
             state.dataSize->DataIsDXCoil = false;
             state.dataSize->DataTotCapCurveIndex = 0;
             state.dataSize->DataFlowUsedForSizing = 0.0;
-        }
-
-        // STEP 3: use the greater of cooling and heating air flow rates for system flow
+            if (this->m_sysType == SysType::PackagedAC) EqSizing.HeatingCapacity = false;
+        }
+
+        // STEP 3A: Find VS cooling coil air flow to capacity ratio and adjust design air flow
+        // this does not use nominal speed level air flow (VarSpeedCoil(WhichCoil).NormSpedLevel), should it?
+        if (this->m_sysType == SysType::PackagedAC || this->m_sysType == SysType::PackagedHP || this->m_sysType == SysType::PackagedWSHP) {
+            Real64 coolingToHeatingCapRatio = 1.0;
+            if ((this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed ||
+                 this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit) &&
+                this->m_MaxCoolAirVolFlow == DataSizing::AutoSize) {
+                int normSpeed = state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex).NormSpedLevel;
+                Real64 coolingAirFlowToCapacityRatio =
+                    state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex).MSRatedAirVolFlowPerRatedTotCap(normSpeed);
+                EqSizing.CoolingAirVolFlow = EqSizing.DesCoolingLoad * coolingAirFlowToCapacityRatio;
+                coolingToHeatingCapRatio = EqSizing.DesCoolingLoad / EqSizing.DesHeatingLoad;
```

Comment:
This is throwing a divide by zero in debug build, that's the one CI failure with PackagedTerminalAirConditionerVSAS.  I'll push up a version of the branch with develop pulled in, but I expect it to be the exact same as before.  Let me know if you want to discuss how to address that condition.

Reply:
This was put in right before these StdRhoAir changes. I was trying to get UnitarySystem to match VS coil sizing. Now this has to be changed to include recent changes. Thanks for the divide by heads up, that will help with the revision.